### PR TITLE
Adding two GCI Mentors

### PR DESCRIPTION
**Pull request by kartikeya23**
#216 
Adding the following mentors:
- Anshuman Verma 
- Vaibhav Singh

Deployment link: http://kartikeya23.github.io/fossasia.org/team.html

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -1269,8 +1273,41 @@ <h1>Developers</h1>
 						<span>Full Stack Developer</span>
 					</div>
 				</div>
-			</div>
-			</div>
+
+				<div class="col-md-3 col-sm-6 speaker-column">
+					<div class="speaker">
+						<div class="image-holder">
+							<img class="background-image" alt="Vibhav Singh" src="img/VaibhavSingh.jpg">
```
Comment (sayak119) [inline team.html]: @kartikeya23 the name in the alt part is wrong. Can you please change it.

**Review by sayak119 — CHANGES_REQUESTED** [review]

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please properly resize images, please see the other images in the repo. :)

File: team.html [hunk 2]
```diff
@@ -1269,8 +1273,41 @@ <h1>Developers</h1>
...
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="https://github.com/vaibhavsingh97"><i class="icon fa fa-github"></i></a>
+									<a href="https://www.facebook.com/profile.php?id=100002720353851&ref=br_rs"><i class="icon social_facebook"></i></a>
+									<a href="https://twitter.com/vaibhavsingh97"><i class="icon social_twitter"></i></a>
+								</div>
+							</div>
+						</div>
+						<span class="speaker-name">Vaibhav Singh</span>
+						<span>GCI Mentor</span>
```
Comment (vaibhavsingh97) [inline team.html]: Change it to Full-Stack Developer / Web Developer

File: team.html [hunk 3]
```diff
@@ -1269,8 +1273,41 @@ <h1>Developers</h1>
...
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="https://github.com/anshumanv"><i class="icon fa fa-github"></i></a>
+									<a href="https://www.facebook.com/anshumanv12"><i class="icon social_facebook"></i></a>
+									<a href="https://twitter.com/Anshumaniac12"><i class="icon social_twitter"></i></a>
+								</div>
+							</div>
+						</div>
+						<span class="speaker-name">Anshuman Verma</span>
+						<span>GCI Mentor</span>
```
Comment (vaibhavsingh97) [inline team.html]: Same

File: team.html [hunk 4]
```diff
@@ -1269,8 +1273,41 @@ <h1>Developers</h1>
...
+
+				<div class="col-md-3 col-sm-6 speaker-column">
+					<div class="speaker">
+						<div class="image-holder">
+							<img class="background-image" alt="Vibhav Singh" src="img/VaibhavSingh.jpg">
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="https://github.com/vaibhavsingh97"><i class="icon fa fa-github"></i></a>
+									<a href="https://www.facebook.com/profile.php?id=100002720353851&ref=br_rs"><i class="icon social_facebook"></i></a>
+									<a href="https://twitter.com/vaibhavsingh97"><i class="icon social_twitter"></i></a>
```
Comment (vaibhavsingh97) [inline team.html]: Can you please add website (https://vaibhavsingh97.me/) and linkedin (https://www.linkedin.com/in/vaibhav-singh/) 😅

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by kartikeya23** [comment]
Made the changes @vaibhavsingh97

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM!

**Comment by kartikeya23** [comment]
@sayak119 and @abishekvashok
I've made the changes, please review once again

**Review by sayak119 — APPROVED** [review]

**Review by ParthS007 — APPROVED** [review]

**Comment by anshumanv** [comment]
Thanks for this @kartikeya23 :smile:

**Pull request merged**